examples/cors/cors_middleware.go

```diff
@@ -0,0 +1,35 @@
+package main
```

Comment:
Could you move it to a new file in `midllewares` package.

Reply:
OK